templates/nginx.conf

```diff
@@ -60,10 +60,7 @@ server {
       # lemmy_port: 8536
 
       set $proxpass "http://0.0.0.0:{{lemmy_ui_port}}";
-      if ($http_accept = "application/activity+json") {
-        set $proxpass "http://0.0.0.0:{{lemmy_port}}";
-      }
-      if ($http_accept = "application/ld+json; profile=\"https://www.w3.org/ns/activitystreams\"") {
+      if ($http_accept ~ "(application\/activity\+json|application\/ld\+json)") {
```

Comment:
`if ($http_accept ~ "application/.*") ` works fine, please use that (already active on ds9.lemmy.ml)